src/main/java/zmaster587/advancedRocketry/dimension/DimensionProperties.java

```diff
@@ -1624,10 +1624,12 @@ public void writeToNBT(NBTTagCompound nbt) {
 		
 		writeTechnicalNBT(nbt);
 	}
-	
-	public int getAverageTemp()
-	{
-		averageTemperature = AstronomicalBodyHelper.getAverageTemperature(this.getStar(), this.getSolarOrbitalDistance(), this.getAtmosphereDensity());
+
+	/**
+	 * @return temperature of the planet in Kelvin
+	 */
+	@Override
+	public int getAverageTemp() {
```

Comment:
The calculation and assignment for averageTemperature was calculated just before this line, is it getting calculated somewhere else now?